test/integration/utilization/system-info.tap.js

```diff
@@ -51,6 +56,7 @@ test('pricing system-info aws', function (t) {
 
     // This will throw an error if the sys info isn't being cached properly
     t.ok(awsRedirect.isDone(), 'should exhaust nock endpoints')
+    t.ok(ecsScope.isDone())
```

Comment:
i know you have unit tests but this integration test should be assertion the ecs vendor:

```
    t.same(systemInfo.vendors.ecs, {
      ecsDockerId: 'ecs-container-1'
    })
```